pkg/quota/util/helper.go

```diff
@@ -0,0 +1,30 @@
+package util
+
+import (
+	"strings"
+
+	apierrs "k8s.io/kubernetes/pkg/api/errors"
+)
+
+// errMessageString is a part of error message copied from quotaAdmission.Admit() method in
+// k8s.io/kubernetes/plugin/pkg/admission/resourcequota/admission.go module
+const errQuotaMessageString = `exceeded quota:`
+const errQuotaUnknownMessageString = `status unknown for quota:`
+const errLimitsMessageString = `exceeds the maximum limit`
+
+// IsErrorQuotaExceeded returns true if the given error stands for a denied request caused by detected quota
+// abuse.
+func IsErrorQuotaExceeded(err error) bool {
+	if isForbidden := apierrs.IsForbidden(err); isForbidden || apierrs.IsInvalid(err) {
```

Comment:
Unit tests are not enough, since you provide the expected values. :+1: for the integration and extended tests which actually verify the interactions with docker.